Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Use AST Base image
-FROM checkmarx/ast-cli:2.3.3@sha256:6bdb2d0b18246ca1a269de2860938313ea14ca2d6e443c953c84fbac9ee5df85
+FROM checkmarx/ast-cli:2.3.5@sha256:af9222f9be2c79738cf2e1d33a0e8da566e879d7
```

Comment:
sha was not changed in the previous releases, are you sure you need to change it?